src/test/addresses_mainnet.sol

```diff
@@ -467,5 +468,12 @@ contract Addresses {
         addr["MCD_CLIP_GNO_A"]                  = 0xd9e758bd239e5d568f44D0A748633f6a8d52CBbb;
         addr["MCD_CLIP_CALC_GNO_A"]             = 0x17b6D0e4237ea7F880aF5F58257cd232a04171D9;
         addr["SPARK_PROXY"]                     = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
```

Comment:
Noting here that this should be `SUBPROXY_SPARK`.

Cleanup from a previous spell technically, but pertinent to current spell as this is declared locally as `SUBPROXY_SPARK` in the main spell file, so I would be amending this in order to line them up as previous spell team (including me) must've overlooked this fix.